packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime-core",
-  "version": "0.0.99",
+  "version": "0.0.991",
```

Comment:
how do I upgrade the prime version after `0.0.99`?

Reply:
`0.0.100` I would think

Reply:
but isn't `0.0.100` < `0.0.99`?

Reply:
0.0.100 is correct.

We will hit 0.1.0 someday.